pupil_src/launchables/player.py

```diff
@@ -531,7 +531,6 @@ def handle_notifications(n):
             p.alive = False
         g_pool.plugins.clean()
 
-        g_pool.capture.cleanup()
```

Comment:
Is `g_pool.capture` part of `g_pool.plugins`? Have we called `capture.cleanup()` all the time? :grimacing: